### PR TITLE
Allow `baseurlpath` to be configured with an env var for when running behind a reverse proxy.

### PR DESCRIPTION
**Pull request by Bwvolleyball**
For cases when we need to run this container behind a reverse proxy - i.e. the reverse proxy provides SSL and terminates before it gets to this container, we need to allow `baseurlpath` to be set by an env var.

This allows us to use `https://my.reverse.proxy` as a host and base url, so that urls are not generated as `http://my.reverse.proxy`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Name|Required/Optional|Description
 `SIMPLESAMLPHP_IDP_ADMIN_PASSWORD`|Optional|The password of admin of this IdP. Default is `secret`.
 `SIMPLESAMLPHP_IDP_SECRET_SALT`|Optional|This is a secret salt used by this IdP when it needs to generate a secure hash of a value. Default is `defaultsecretsalt`.
 `SIMPLESAMLPHP_IDP_SESSION_DURATION_SECONDS`|Optional|This value is the duration of the session of this IdP in seconds.
+`SIMPLESAMLPHP_BASE_URL_PATH`|Optional|This value allows you to override the base URL. Valuable for setting an `https://` base url behind a reverse proxy. **IMPORTANT** This url _must_ still end with `/simplesaml/` example: `https://my.proxy.com/simplesaml/` [Read more here](https://github.com/kenchan0130/docker-simplesamlphp/blob/master/config/simplesamlphp/config.php#L9-L24). Default is `simplesaml/`.
```
Comment (kenchan0130) [inline README.md:67]: 1. I would like the names of the environment variables to be names that identify them as belonging to the IdP, such as `SIMPLESAMLPHP_IDP_BASE_URL_PATH`.
2. Referencing a URL with a specified number of lines makes it difficult to track when code is added or removed.
Therefore, if you wish to add a link to an environment variable description, please keep it to the file name.
3. Constraints such as the below can increase end-user difficulty.
Therefore, I think we should specialize in specifying only the base URL, excluding the path part.
    
    >  **IMPORTANT** This url _must_ still end with `/simplesaml/` example: `https://my.proxy.com/simplesaml/`

Comment (kenchan0130) [inline README.md:67]: 1. I would like the names of the environment variables to be names that identify them as belonging to the IdP, such as `SIMPLESAMLPHP_IDP_BASE_URL_PATH`.
2. Referencing a URL with a specified number of lines makes it difficult to track when code is added or removed.
Therefore, if you wish to add a link to an environment variable description, please keep it to the file name.
3. Constraints such as the below can increase end-user difficulty.
Therefore, I think we should specialize in specifying only the base URL, excluding the path part.
    
    >  **IMPORTANT** This url _must_ still end with `/simplesaml/` example: `https://my.proxy.com/simplesaml/`

Comment (Bwvolleyball) [reply README.md]: How does this look now @kenchan0130?

**Review by kenchan0130 — APPROVED** [review]
@Bwvolleyball Thank you for the multiple corrections.
LGTM!

**Pull request merged**